src/main/resources/lang/default.lang

```diff
@@ -2323,6 +2335,7 @@ types:
 	material: material¦s @a
 	itemstack: item stack¦s @an
 	itementity: dropped item¦s @a # same as entities.dropped item.name
+	itemflag: item flag¦s @an
```

Comment:
should be `itemflags:` or changed in bukkit classes to `itemflag`

Reply:
probably the latter is better, seems to be how everything else is done, thanks for pointing that out